File: Evergrande/src/gestorAplicación/Agente.java
package gestorAplicación;

public class Agente {
	private int cedula;
	private String nombreCompleto;
	private Vehiculo vehiculo;
	private String placaVehiculo;
	private String telefonoFijo;
	private String telefonoCelular;
	
	public int getCedula() {
		return cedula;
	}
	
	public void setCedula(int cedula) {
		this.cedula = cedula;
	}
	
	public String nombreCompleto() {
		return nombreCompleto;
	}
	
	public void setNombreCompleto(String nombreCompleto) {
		this.nombreCompleto = nombreCompleto;
	}
	
	public Vehiculo getvehiculo() {
		return vehiculo;
	}
	
	public void setVehiculo(Vehiculo vehiculo) {
		this.vehiculo = vehiculo;
	}
	
	public String getPlacaVehiculo() {
		return placaVehiculo;
	}
	
	public void setPlacaVehiculo(String placa) {
		this.placaVehiculo = placa;
	}
	
	public String getTelefonoFijo() {
		return telefonoFijo;
	}
	
	public void setTelefonoFijo(String telefonoFijo) {
		this.telefonoFijo = telefonoFijo;
	}
	
	public String getTelefonoCelular() {
		return telefonoCelular;
	}
	
	public void setTelefonoCelular(String telefonoCelular) {
		this.telefonoCelular = telefonoCelular;
	}
}

File: Python/src/gestorAplicacion/herencia/Agente.py
from Persona import Persona

class Agente(Persona):
  AGENTES = []

  def __init__(self, cedula, nombreCompleto, telefonoFijo, telefonoCelular, vehiculo, placaVehiculo):
    super().__init__(cedula, nombreCompleto, telefonoFijo, telefonoCelular)
    self.vehiculo = vehiculo
    self.placaVehiculo = placaVehiculo
  
  def getVehiculo(self):
    return self.vehiculo
  
  def setVehiculo(self, nVehiculo):
    self.vehiculo = nVehiculo
  
  def getPlaca(self):
    return self.placa
  
  def setPlaca(self, nPlaca):
    self.placa = nPlaca


  # def buscarAgente(self, id)
